neo/Network/P2P/RemoteNode.cs

```diff
@@ -224,7 +224,7 @@ protected override void PostStop()
 
         internal static Props Props(NeoSystem system, object connection, IPEndPoint remote, IPEndPoint local)
         {
-            return Akka.Actor.Props.Create(() => new RemoteNode(system, connection, remote, local)).WithMailbox("remote-node-mailbox");
+            return Akka.Actor.Props.Create(() => new RemoteNode(system, connection, remote, local, new ProtocolActorFactory(Context))).WithMailbox("remote-node-mailbox");
```

Comment:
@lock9, I think that this will be very good for better organization of the mailbox!